st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -130,6 +173,45 @@ def get(self, id, attribute, **kwargs):
         return result
 
 
+class ActionExecutionReRunController(ActionExecutionsControllerMixin, ResourceController):
+    supported_filters = {}
+    exclude_fields = [
+        'result',
+        'trigger_instance'
+    ]
+
+    class ExecutionParameters(object):
```

Comment:
This is a "limitation" of our existing `jsexpose` decorator - in most other places we can just use DB models, but in this case API only takes dictionary with "parameters" key so we need to create a "dummy" class for that.